README.md

```
@@ -1,3 +1,708 @@
+
+### Linting
+
+Run ESLint:
+
+```bash
+npm run lint
+```
+
+## Code Structure
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)